### ISSUE TITLE
📣 Call for maintainers

### ISSUE THREAD

**Issue by jkimbo** [graphql-python/graphene-django#1049]
Unfortunately I no longer have the time to maintain Graphene-Django so I'm putting out this call for anyone who wants to get involved with development. Ideally you'll already have contributed to this library in some way or another.

@zbyte64 has already said that he's willing to step in but is anyone else interested?

**Comment by tcleonard** [comment]
I have started to use graphene-django quite extensively recently and would not mind being involved in this awesome project.
I am far from having a good understanding of all what graphene(-django) can offer but I'm happy to offer my modest help.

**Comment by ulgens** [comment]
Hey @jkimbo , i was meant to give some of my time to project before but i couldn't manage it that time. Maybe, this time is the time for me to try again :slightly_smiling_face:

**Comment by jkimbo** [comment]
Thanks @tcleonard and @ulgens ! Please get in contact with @zbyte64 to organise. He has admin access on the repo so should be able to handle any problems.

Here are some hand over notes and areas that need work:

* [v3 of Graphene-Django](https://github.com/graphql-python/graphene-django/issues/705) is waiting on Graphene v3 to be released. [Graphene v3](https://github.com/graphql-python/graphene/issues/1127) is ready to go but is being held up because the other dependant libraries (graphene-mongo etc) have not been updated. There is no ETA on when they will be updated or when Graphene v3 will be released.
* The biggest missing feature from the v3 release currently is dataloaders. The new graphql-core version dropped support for Promises so you have to use asyncio to achieve the same thing. However the Django ORM does not currently support asyncio so it's unclear what to do until it does.
* The master branch of Graphene-Django is for the v2 releases. There is a v3 branch which should be periodically updated with the changes in master. Once v3 has a clear path to be released the v3 branch should be merged into master. Maintaining a v2 branch to release bugfixes is probably a good idea as well.
* The release process is as follows:
	* Update the version in https://github.com/graphql-python/graphene-django/blob/master/graphene_django/__init__.py and commit it
    * Create a new GitHub release with the new version number
	* GitHub will create a new tag and GitHub actions will push a new release to PyPi
* The biggest source of confusion currently is the fact that Graphene-Django defaults a lot to Relay patterns when most developers don't use Relay (see https://github.com/graphql-python/graphene-django/issues/349, https://github.com/graphql-python/graphene-django/issues/957, https://github.com/graphql-python/graphene-django/issues/274 and probably others). This should really be addressed and IMO the Relay integration should be completely optional. I've written up some thoughts here: https://github.com/graphql-python/graphene-django/issues/957#issuecomment-626307802 Possibly a good opportunity do it with the v3 release.
* The https://github.com/eamigo86/graphene-django-extras library is very good and should probably be integrated

Sorry it's quite rough. Feel free to ask me questions either here or on Slack.

✌️ out

**Comment by zbyte64** [comment]
I will be regularly available for this project every Friday @ 3:00pm-4:00pm PST. If you show up on slack or twitch I'll be available to discuss or help with any issues related to Django graphene you might be having.

**Comment by asencis** [comment]
Hey @zbyte64 I'm looking to commit to Django-graphene from the Django REST framework - are you still accepting new applications for contributors/maintainers? From my own experiences with the library - I'd like to help in the area of documentation but also really happy to help with any modest-sized PRs that are needed.

**Comment by jzingh98** [comment]
@zbyte64 I'm a relative newcomer in graphene-django world but would love to get involved. As a beginner, the primary pain points I've had to overcome relate to poor/vague documentation. Would definitely like to help make those improvements while that pain is fresh in my mind haha

**Comment by zbyte64** [comment]
Hi @asencis ; would love to have you join the team. Please join us on slack.

**Comment by iltoningui** [comment]
I've recently started building opensource tools and working with django and I would like to be involved in the project.

**Comment by kevindice** [comment]
Hi @zbyte64 , I'd also be interested.

**Comment by yaserrahimi** [comment]
Hi @zbyte64 how i can start?

**Comment by jackton1** [comment]
@zbyte64 I'll like to also get involved, currently actively using graphene-django in production.

**Comment by allangz** [comment]
Hi guys, I also would like to help.

**Comment by Luferov** [comment]
Hi! Me too! Could I get link to slack Chunnel?

**Comment by taekyun-quriously** [comment]
me too! can i get link to slack channel? my email address is taxijjang@gmail.com

**Comment by hiporox** [comment]
@bellini666 @Luferov @taekyun-quriously @allangz There is a link to the [graphene slack channel](https://join.slack.com/t/graphenetools/shared_invite/enQtOTE2MDQ1NTg4MDM1LTA4Nzk0MGU0NGEwNzUxZGNjNDQ4ZjAwNDJjMjY0OGE1ZDgxZTg4YjM2ZTc4MjE2ZTAzZjE2ZThhZTQzZTkyMmM) in the `README`, I have also added it here. There is a `django` channel in there

@zbyte64 can confirm if that is the correct location, but that is my guess as someone who is also interested in helping out

**Comment by Luferov** [comment]
@hiporox thank you so much  i found and joined to slack channel a few days ago.

**Comment by elyas-hedayat** [comment]
@jazzband Could you help to maintain this repo?
